packages/playwright-core/src/server/javascript.ts

```diff
@@ -328,3 +328,19 @@ export class JavaScriptErrorInEvaluate extends Error {
 export function isJavaScriptErrorInEvaluate(error: Error) {
   return error instanceof JavaScriptErrorInEvaluate;
 }
+
+export function sparseArrayToString(entries: { index: number, value: any }[]): string {
```

Comment:
Why don't we accept `{ name, value }[]` array? Then cr/wk wouldn't need to additionally convert, since they share the same format.